setup.py

```diff
@@ -0,0 +1,36 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="object_store",
+    url="https://github.com/xlab-si/python-object-store-library",
+    version="0.0.2",
+    author="XLAB d.o.o.",
+    author_email="pypi@xlab.si",
+    packages=['object_store'],
+    description="Python library for different object storages that have common methods.",
+    long_description="file: README.md",
+    long_description_content_type="text/markdown",
+    license="Apache Software License",
+    license_file="LICENSE",
+    classifiers=[
+        "Development Status:: 2 - Pre - Alpha",
+        "Environment:: Console",
+        "Intended Audience:: Developers",
+        "Intended Audience:: System Administrators",
+        "License:: OSI Approved:: Apache Software License",
+        "Operating System:: OS Independent",
+        "Programming Language:: Python",
+        "Programming Language:: Python:: 3",
+        "Programming Language:: Python:: 3.6",
+        "Programming Language:: Python:: 3.7"
+    ],
+    install_requires=[
+        'minio==5.0.4',
+        'boto3==1.10.9',
+        'azure==4.0.0',
+        'azure-storage==0.36.0',
+        'azure-functions==1.0.7']
```

Comment:
Functions here as well.

Reply:
Thank you for noticing. This was removed as well.